### ISSUE TITLE
Still have bug with "adjust startcol by textEdit of CompletionItem"

### ISSUE THREAD

**Issue by iamcco** [neoclide/coc.nvim#1805]
## Result from CocInfo

``` markdown
## versions

vim version: NVIM v0.5.0-nightly-349-ge5da4fd55
node version: v10.16.0
coc.nvim version: 0.0.78
term: xterm-kitty
platform: darwin

## Output channel: git

Looking for git in: git
> git rev-parse --show-toplevel
fatal: not a git repository (or any of the parent directories): .git

Error 128 on: 'git rev-parse --show-toplevel' in /Users/aioiyuuko/workspace/pxn
fatal: not a git repository (or any of the parent directories): .git
> git rev-parse --show-toplevel
> git rev-parse --show-toplevel
> git --no-pager blame -b -p --root -L1,1 --date relative nvim/coc-settings.json
resolved root: /Users/aioiyuuko/.configrc
> git ls-files -- nvim/coc-settings.json
resolved root: /Users/aioiyuuko/.configrc
> git symbolic-ref --short HEAD
> git diff --name-status
> git diff --staged --name-status
> git ls-files --others --exclude-standard
> git --no-pager show :nvim/coc-settings.json
> git diff /Users/aioiyuuko/.configrc/nvim/coc-settings.json

## Output channel: watchman

[Info  - 12:37:18 PM] watchman watching project: /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/.eslintr{c.js,c.yaml,c.yml,c,c.json}" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/.eslintignore" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/package.json" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/stylelint.config.js" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/.stylelintrc" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/.stylelintrc.js" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/package.json" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/[tj]sconfig.json" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/package.json" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/{.prettierrc,.prettierrc.json,.prettierrc.yaml,.prettierrc.yml,.prettierrc.js,package.json,prettier.config.js}" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/*.{ts,tsx}" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:18 PM] subscribing "**/*.{js,jsx}" in /Users/aioiyuuko/workspace/pxn
[Info  - 12:37:21 PM] watchman watching project: /Users/aioiyuuko/.configrc
[Info  - 12:37:21 PM] subscribing "**/*.json" in /Users/aioiyuuko/.configrc

## Output channel: leetcode

[State]: State init from: /Users/aioiyuuko/.config/coc/extensions/coc-leetcode-data/state
[statusBar]: enter 2
[statusBar]: enter 1

## Output channel: prettier


## Output channel: snippets

[Error 2020-4-23] parser error: 4

## Output channel: explorer


## Output channel: tailwindcss-language-server


## Output channel: tsserver

[Info  - 12:37:17 PM] Forking TSServer
PATH: /Users/aioiyuuko/go/bin:/Users/aioiyuuko/.rbenv/shims:/Users/aioiyuuko/google-cloud-sdk/bin:/Users/aioiyuuko/.pub-cache/bin:/Users/aioiyuuko/sdk/flutter/bin/cache/dart-sdk/bin:/Users/aioiyuuko/sdk/flutter/bin:/Users/aioiyuuko/.config/npm/bin:/Users/aioiyuuko/Library/Android/sdk/platform-tools:/Users/aioiyuuko/Library/Android/sdk/tools:/Library/Java/JavaVirtualMachines/jdk-11.jdk/Contents/Home/bin:/Users/aioiyuuko/.pyenv/shims:/usr/local/opt/mysql@5.7/bin:/Users/aioiyuuko/.fzf/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/kitty.app/Contents/MacOS 
[Info  - 12:37:17 PM] Started TSServer
{
  "path": "/Users/aioiyuuko/.config/coc/extensions/node_modules/coc-tsserver/node_modules/_typescript@3.8.3@typescript/lib",
  "_pathLabel": "",
  "_api": {
    "versionString": "3.8.3",
    "version": "3.8.3"
  }
}

## Output channel: eslint

[Info  - 12:37:18 PM] ESLint server running in node v10.16.0

## Output channel: highlight

[Info  - 12:37:18 PM] Highlight server running in node v10.16.0

```

## Describe the bug

CompleteItem disappear when activate `coc-postfix`

## Reproduce the bug

Install extension: `coc-tsserver`, `coc-postfix`

open file `t.ts`:

``` typescript
class FormGroup {

}

class A {
  constructor() {
    this.form = new f|
  }
}

```

`|` is cursor position，when `coc-postfix` activate, I can not get the `FormGroup` autocomplete item.
If deactivate `coc-postfix`, it works fine.

## Screenshots (optional)

deactivate `coc-postfix`
![image](https://user-images.githubusercontent.com/5492542/80060082-39206380-8560-11ea-9142-c2388b05a8d5.png)

activate `coc-postfix`
![image](https://user-images.githubusercontent.com/5492542/80060105-49d0d980-8560-11ea-95fc-95df25e433d6.png)

**Comment by fannheyward** [comment]
I'll test this with `coc-postfix`.

**Comment by chemzqm** [comment]
`coc-postfix` could provide textEdit on completion resolve to avoid change of completion position.

**Comment by fannheyward** [comment]
@iamcco just released `coc-postfix` 0.0.8, with a quick fix for this issue.

**Comment by iamcco** [comment]
@fannheyward works fine after update 👍
